src/pcap/devtools/mock-bosh-setup/ca.go

```diff
@@ -0,0 +1,202 @@
+package main
+
+import (
+	"bytes"
+	"crypto/rand"
+	"crypto/rsa"
+	"crypto/x509"
+	"crypto/x509/pkix"
+	"encoding/pem"
+	"io"
+	"math/big"
+	"os"
+	"time"
+)
+
+// Some helper tools to generate x509 CAs and Certificates
```

Comment:
This is not a lot of documentation for this much code.

A few godoc comments describing what the specific stuct or function do would be great. A lot of it is self-explaining, fair enough.

For the certificate creation (newCA, ClientCert), some of the key points would be helpful, e.g. "uses the provided CA to create a new client certificate. The appropriate usage flags for client authentication and signature are set."